koupleless-ext/koupleless-build-plugin/koupleless-base-build-plugin/src/main/resources/adapter-mapping-default.yaml

```
@@ -7,129 +7,172 @@ adapterMappings:
-      groupId: com.alipay.sofa.koupleless
\ No newline at end of file
+      groupId: com.alipay.sofa.koupleless
+      version: 1.2.4
+  - matcher:
+      groupId: org.springframework.boot
+      artifactId: spring-boot
+      versionRange: "[2.3.*]"
+    adapter:
+      artifactId: koupleless-adapter-springboot-2.3
+      groupId: com.alipay.sofa.koupleless
+      version: 1.2.2
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)